### ISSUE TITLE
I found an official bug, what is the reason for this or what is the error?(Served eth_blockNumber)

### ISSUE THREAD

**Issue by zrock23** [ethereum/go-ethereum#23192]
Windows 10 pro
Geth 1.10.4 stable

```
starting node command: geth.exe --mine --miner.notify.full --allow-insecure-unlock --unlock "0x63A34e73F622e3b0230381563D3eea6f1b3c32b1,0x69C130e8D69B8662ABb82dd213e659994b995576,0xC32133BFb3bbC1fB61F4478bC5fC0d9bFfCEbb82" --syncmode full --datadir "C:\Geth Node\data\data" --nat upnp --maxpeers 150 --cache.trie.rejournal 30m0s --cache.preimages --bloomfilter.size 10024 --rpc.allow-unprotected-txs --snapshot console
###############
Аfter starting wallet the following error appears: (WARN [07-11|01:11:54.865|rpc/handler.go:306]             Served eth_blockNumber                   conn=127.0.0.1:53827 reqid=1 t=0s err="the method eth_blockNumber does not exist/is not available")
##############
```

![bug](https://user-images.githubusercontent.com/70485940/125177456-84b24200-e1e4-11eb-83fd-659653ff0127.jpg)

**Comment by karalabe** [comment]
That's not the command you used in the screenshot above. In your logs you are running with the `http` RPC endpoint open but it was not specified in the startup command you pasted here.

Btw, the error most probably is that you specified which namespaces to expose on the HTTP interface and did not specify `eth`.

**Comment by karalabe** [comment]
Your CORS is also looking pretty wild. Are you sure you want all those things to access you node?

**Comment by zrock23** [comment]
Thanks! 🤗 I will take this recommendation!Вземете Outlook за AndroidОт: Péter Szilágyi ***@***.***>Изпратено: вторник, 13 юли 2021 г., 15:52 ч.До: ethereum/go-ethereumЯк: zrock23; AuthorТема: Re: [ethereum/go-ethereum] I found an official bug, what is the reason for this or what is the error?(Served eth_blockNumber) (#23192)
Your CORS is also looking pretty wild. Are you sure you want all those things to access you node?

**Comment by karalabe** [comment]
Unsure if this solved it or not. Please double and triple check your command, it is definitely off. If you specify explicitly what API endpoints to expose, make sure that `eth` is in there if you want to access it. Also you might want to check your flags from the above list, it seems a lot of them are redundant and just the defaults.

**Comment by zrock23** [comment]
I made some adjustments prematurely, I noticed that the symbol = and " " represent links for stability of the links. I added some settings like --miner.threads = 1 and --miner.noverify and the startup seems to be fine ... I did a few transactions and everything is fine. I was more interested in the "p2p" mode but at the moment I have no more ideas.
![example2](https://user-images.githubusercontent.com/70485940/125813318-77354613-8b7c-4995-aa17-0fe51be8bf1b.png)
